monai/networks/utils.py

```diff
@@ -654,32 +657,35 @@ def convert_to_onnx(
                 del kwargs["example_outputs"]
             mode_to_export = torch.jit.script(model, **kwargs)
 
+        if torch.is_tensor(inputs):
+            inputs = (inputs,)
```

Comment:
Is this one necessary? The input parameter in line 670 of the torch.onnx.export function has already been converted through `tuple` function.

Thanks  
Bin

Reply:
If you apply tuple() to tensor, it produces a tuple from it - iterating over and removing one dimension. onnx.export() does accept bare tensor as input in case of single input, so this code makes this method do the same.